static/state_bydate.js

```diff
@@ -396,6 +394,47 @@ function parsingDone(results, file) {
   });
 }
 
+var groupToEssentialMap = new Map();
+
+var groupMappings = [
+  {groupName:"Airport",essential:true},
+  {groupName:"Alcohol",essential:true},
+  {groupName:"Arts & Entertainment",essential:false},
+  {groupName:"Banks",essential:true},
+  {groupName:"Beach",essential:false},
+  {groupName:"Big Box Stores",essential:false},
+  {groupName:"Bus",essential:true},
+  {groupName:"Colleges & Universities",essential:false},
+  {groupName:"Convenience Store",essential:true},
+  {groupName:"Discount Stores",essential:false},
+  {groupName:"Drug Store",essential:true},
+  {groupName:"Fast Food Restaurants",essential:true},
+  {groupName:"Fitness Center",essential:false},
+  {groupName:"Food",essential:true},
+  {groupName:"Gas Stations",essential:true},
+  {groupName:"Government",essential:true},
+  {groupName:"Grocery",essential:true},
+  {groupName:"Hardware Stores",essential:true},
+  {groupName:"Hotel",essential:false},
+  {groupName:"Medical",essential:true},
+  {groupName:"Nightlife Spots",essential:false},
+  {groupName:"Office",essential:false},
+  {groupName:"Outdoors & Recreation",essential:false},
+  {groupName:"Professional & Other Places",essential:false},
+  {groupName:"Residences",essential:true},
```

Comment:
this reminded me to ask Foursquare about whether one's own home counts as a Residence visit.  People going to other people's houses will be an interesting thing to track, regardless of whether we call it "essential" or not